### PR TITLE
Add SEND_TO_INGRESS port table in schema.h

### PR DESCRIPTION
**Pull request by baxia-lan**
HLD: https://github.com/sonic-net/SONiC/blob/master/doc/pins/send_to_ingress_hld.md

### CODE REVIEW

**Comment by baxia-lan** [comment]
@prsunny PTAL

**Comment by baxia-lan** [comment]
@mint570

PTAL

File: common/schema.h [hunk 1]
```diff
@@ -271,6 +272,9 @@ namespace swss {
 #define CFG_PORT_TABLE_NAME           "PORT"
 #define CFG_PORT_CABLE_LEN_TABLE_NAME "CABLE_LENGTH"
 
+#define CFG_SEND_TO_INGRESS_PORT_TABLE_NAME  "SEND_TO_INGRESS_PORT"
+#define SEND_TO_INGRESS_PORT_NAME            "SEND_TO_INGRESS"
```
Comment (prsunny) [inline common/schema.h:276]: Please prefix this with CFG_. Also is this a new table in Config_DB? if so, what is the above one for?

Comment (baxia-lan) [reply common/schema.h]: Removed this port name.
It was used as hardcoded send_to_ingress port name.

**Review by prsunny — APPROVED** [review]

**Comment by wen587** [comment]
Hi @baxia-lan , could you add YANG model for SEND_TO_INGRESS_PORT?

**Pull request merged**